### PR TITLE
ER-470 - Updating DownloadCount on download and minor tweaks.

### PR DESCRIPTION
**Pull request by chrisjensenuk**
OK so I said the last PR was the final PR for reporting however it appears not.

- Incrementing the download count every time a report is downloaded
- Minor tweaks to align with the story

### CODE REVIEW

File: src/Provider/Provider.Web/Orchestrators/Reports/ReportDashboardOrchestrator.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 using Esfa.Recruit.Vacancies.Client.Domain.Extensions;
 using Esfa.Recruit.Vacancies.Client.Infrastructure.Client;
+using Microsoft.CodeAnalysis.Operations;
```
Comment (shomavg) [inline src/Provider/Provider.Web/Orchestrators/Reports/ReportDashboardOrchestrator.cs]: is this namespace being used?

File: src/Provider/Provider.Web/ViewModels/Reports/ProviderApplicationsReport/ProviderApplicationsReportCreateViewModel.cs [hunk 2]
```diff
@@ -1,4 +1,7 @@
-﻿namespace Esfa.Recruit.Provider.Web.ViewModels.Reports.ProviderApplicationsReport
+﻿using System.Collections;
```
Comment (shomavg) [inline src/Provider/Provider.Web/ViewModels/Reports/ProviderApplicationsReport/ProviderApplicationsReportCreateViewModel.cs]: This namespace could be removed.

File: src/Provider/Provider.Web/Orchestrators/Reports/ReportDashboardOrchestrator.cs [hunk 3]
```diff
@@ -50,11 +53,15 @@ public async Task<ReportDownloadViewModel> GetDownloadCsvAsync(long ukprn, Guid
 
             _vacancyClient.WriteReportAsCsv(stream, report);
 
-            return new ReportDownloadViewModel
+            var vm = new ReportDownloadViewModel
```
Comment (shomavg) [inline src/Provider/Provider.Web/Orchestrators/Reports/ReportDashboardOrchestrator.cs]: Do we need the vm variable?

File: src/Provider/Provider.Web/Orchestrators/Reports/ReportDashboardOrchestrator.cs (hunk 1 above)
Comment (chrisjensenuk) [reply src/Provider/Provider.Web/Orchestrators/Reports/ReportDashboardOrchestrator.cs]: 👍 nope

File: src/Provider/Provider.Web/ViewModels/Reports/ProviderApplicationsReport/ProviderApplicationsReportCreateViewModel.cs (hunk 2 above)
Comment (chrisjensenuk) [reply src/Provider/Provider.Web/ViewModels/Reports/ProviderApplicationsReport/ProviderApplicationsReportCreateViewModel.cs]: 👍

File: src/Provider/Provider.Web/Orchestrators/Reports/ReportDashboardOrchestrator.cs (hunk 3 above)
Comment (chrisjensenuk) [reply src/Provider/Provider.Web/Orchestrators/Reports/ReportDashboardOrchestrator.cs]: 👍

**Pull request merged**